solidity/test/bridge/Bridge.Parameters.test.ts

```diff
@@ -12,12 +12,12 @@ describe("Bridge - Parameters", () => {
   let governance: SignerWithAddress
   let thirdParty: SignerWithAddress
   let bridge: Bridge & BridgeStub
+  let bridgeGovernance: BridgeGovernance
 
   before(async () => {
     // eslint-disable-next-line @typescript-eslint/no-extra-semi
-    ;({ governance, thirdParty, bridge } = await waffle.loadFixture(
-      bridgeFixture
-    ))
+    ;({ governance, thirdParty, bridge, bridgeGovernance } =
+      await waffle.loadFixture(bridgeFixture))
   })
 
   describe("updateDepositParameters", () => {
```

Comment:
Since we've got `Bridge.Governance.test.ts` that already covers parameters upgrades, this file seems to duplicate the other tests. We cannot call `updateDepositParameters` and others directly. Maybe we could modify tests in this file to just verify that non-governance cannot call the update functions. Similar as in https://github.com/keep-network/keep-core/blob/main/solidity/ecdsa/test/WalletRegistry.Parameters.test.ts